### PR TITLE
Commerce 6577 Update PayPal SDK + minor fixes

### PR DESCRIPTION
**Pull request by luca-pellizzon**

### CODE REVIEW

**Comment by luca-pellizzon** [comment]
ci:test:sf

**Comment by luca-pellizzon** [comment]
ci:test:relevant

File: modules/apps/commerce/commerce-payment-method-paypal/src/main/java/com/liferay/commerce/payment/method/paypal/internal/PayPalCommercePaymentMethod.java [hunk 1]
```diff
@@ -838,22 +848,116 @@ public CommercePaymentResult voidTransaction(
...
 	}
 
-	private RefundRequest _buildRefundRequestBody(
-		String amount, String currencyCode) {
+	private List<PurchaseUnitRequest> _buildFullRequestBody(
+			CommerceOrder commerceOrder, Locale locale)
+		throws PortalException {
 
-		Money money = new Money();
+		List<PurchaseUnitRequest> purchaseUnits = new ArrayList<>();
```
Comment (dsitu) [inline modules/apps/commerce/commerce-payment-method-paypal/src/main/java/com/liferay/commerce/payment/method/paypal/internal/PayPalCommercePaymentMethod.java:855]: We should rename to this.
```suggestion
		List<PurchaseUnitRequest> purchaseUnitRequests = new ArrayList<>();
```

**Review by dsitu — CHANGES_REQUESTED** [review]

**Comment by michaelhashimoto** [comment]
@luca-pellizzon - Some of the builds were aborted in this pull request test. Please retest if results needed. Sorry for any inconvenience.

**Comment by luca-pellizzon** [comment]
ci:test:sf

**Comment by luca-pellizzon** [comment]
ci:test:relevant

**Comment by riccardo-alberti** [comment]
ci:forward

**Pull request closed without merging**